README.md

```diff
@@ -94,6 +94,11 @@ Links to all data sources are including in [Downloads](downloads)
 - see [Ages of confirmed cases 2020-2021]((https://github.com/djay/covidthailand/wiki/cases_ages_all.png))
 - Source: [API: Daily reports of COVID-19 infections](https://data.go.th/dataset/covid-19-daily)
 
+## Cases by Non-Thai Nationality
```

Comment:
I think its clearer to say "Non Thai Cases" or "Cases in Non-Thai". Maybe don't need Nationality word at all?